apps/omg_watcher/test/support/exit_processor/test_helper.ex

```diff
@@ -159,4 +176,21 @@ defmodule OMG.Watcher.ExitProcessor.TestHelper do
 
     long
   end
+
+  @doc """
+  Generates a given length of bytes for the given integer.
+  """
+  @spec binary_from_integer(integer(), integer()) :: binary()
+  def binary_from_integer(integer, num_bytes) do
+    num_bits = num_bytes * 8
+    <<integer::size(num_bits)>>
+  end
+
+  @doc """
+  Generates a certain length of random bytes. Not recommended for identifiers as uniqueness not guaranteed.
+  """
+  @spec random_bytes(integer()) :: binary()
```

Comment:
Oh, no please remove - IMHO it gains nothing

Reply:
yes please